### PR TITLE
Fix 404 layout b4 stable

### PR DESCRIPTION
**Pull request by eri-trabiccolo**

### CODE REVIEW

File: assets/front/scss/0_1_base/_mixins.scss [hunk 1]
```diff
@@ -470,30 +470,36 @@
...
-// This will be not needed anymore with the flex :D
-@mixin make-grid-smart-rows( $items: $grid-columns, $offset : 0, $breakpoints: $grid-breakpoints ) {
-  @each $breakpoint in map-keys($breakpoints) {
+//RTL offset-* classes
+//inspired by the bootstrap offset-* class generator, used to make offset-* classes rtl compliant
+@mixin make-col-offset-rtl($size, $columns: $grid-columns) {
+  $num: $size / $columns;
+  margin-right: if($num == 0, 0, percentage($num));
+  margin-left: auto;
+}
```
Comment (ghost) [inline assets/front/scss/0_1_base/_mixins.scss:480]: cool tnx

Comment (eri-trabiccolo) [reply assets/front/scss/0_1_base/_mixins.scss:480]: that part is copied from bs :P 
I just replaced margin-left with margin-right and added margin-left: auto; to override the default offset-* :D

Comment (ghost) [reply assets/front/scss/0_1_base/_mixins.scss:480]: who said that copying ( and adapting ) good open source code was bad ?... ;) :D

Comment (eri-trabiccolo) [reply assets/front/scss/0_1_base/_mixins.scss:480]: Nobody :D Just didn't want to take different credits besides the ones that I already deserve, they're already a lot :P

Comment (ghost) [reply assets/front/scss/0_1_base/_mixins.scss:480]: ah ah ah
well, I agree :)

**Pull request merged**